sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -622,7 +622,12 @@ class SubquerySuite extends QueryTest with SharedSQLContext {
 
   test("SPARK-15370: COUNT bug with attribute ref in subquery input and output ") {
     checkAnswer(
-      sql("select l.b, (select (r.c + count(*)) is null from r where l.a = r.c) from l"),
+      sql(
```

Comment:
In this test, the query is not a valid query as its missing group by. It used to pass before as we implicitly added r.c into aggregate grouping as part of pulling up correlated predicates in analyzer.